dist/READMEopenshiftdevpreview.md

```diff
@@ -107,8 +107,8 @@ At this point ovn is providing networking for the cluster.
 
 There is a single docker image that is used in all of the ovn daemonsets.
 All daemonset yaml files must be edited to reference the same desired image.
-The images can be found in the official OKD repositories or they can be
-built in the openvswitch/ovn-kubernetes git repo.
+The images can be found in docker.io, or the official OKD repositories or
```

Comment:
``docker.io, the official OKD repositories`` (you have an extra or)

Reply:
Changed